docs/source/index.rst

```diff
@@ -71,3 +71,64 @@ Requirements
 * pycosat
 * pyyaml
 * requests
+
+What's new in conda 4.1.0?
```

Comment:
Kale, feel free to edit out the .0 and merge whenever you are ready!

On Monday, June 27, 2016, Kale Franz <notifications@github.com> wrote:

> In docs/source/index.rst
> <https://github.com/conda/conda-docs/pull/304#discussion_r68573922>:
>
> > @@ -71,3 +71,64 @@ Requirements
> >  * pycosat
> >  * pyyaml
> >  * requests
> > +
> > +What's new in conda 4.1.0?
>
> Let's just make this 4.1. We're releasing 4.1.4 today. The "What's new"
> section is really about new features, and that doesn't change between
> 4.1.0 and 4.1.4+.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda/conda-docs/pull/304/files/a7ebd4bd5fe76b32bb4ae13408dce8274cd29c57#r68573922>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEXgNWdxAJOAXe0ILzk2CC37pVtbX7Kks5qP82FgaJpZM4I9Ktq>
> .
>


-- 
_______________________________

Kerry Watson
Documentation manager

[image: Continuum Analytics] <http://continuum.io>